bindings/rust/bench/benches/handshake.rs

```diff
@@ -28,10 +31,43 @@ pub fn bench_handshake(c: &mut Criterion) {
     bench_handshake_for_libraries! {
         ("s2n-tls", S2NHarness),
         ("rustls", RustlsHarness),
+        ("openssl", OpenSslHarness),
     }
 
     group.finish();
 }
 
-criterion_group!(benches, bench_handshake);
+pub fn bench_handshake_key_exchange(c: &mut Criterion) {
```

Comment:
This macro is getting pretty hard to read, especially with the repeated tokens. I think I'd prefer something a bit simpler along the lines of 
```
for ec_group in [...] {
    bench_group = bench_group(ec_group)
    add_bench_function<S2NHarness>(bench_group);
    add_bench_function<RustlsHarness>(bench_group);
    add_bench_function<OpenSslHarness>(bench_group);
}
```
Not sure whether that syntax works, but something along those lines should be valid?